lib/models/ObjectMD.ts

```diff
@@ -227,12 +227,12 @@ export default class ObjectMD {
                 role: '',
                 storageType: '',
                 dataStoreVersionId: '',
-                isNFS: null,
+                isNFS: undefined,
             },
             dataStoreName: '',
             originOp: '',
-            deleted: false,
-            isPHD: false,
+            deleted: undefined,
+            isPHD: undefined,
```

Comment:
we should esp. double check how these fields are used, and if we don't have mongodb queries/conditions on `delete: false` or `isPHD: false`

Reply:
`deleted: false`: To be retro-compatible, we always use to conditions: either delete doesn't exist or it's equal to false.
`isPHD: false`: We don't have mongo checks on this.
Conditions in the code also work when the fields are undefined.